include/picongpu/param/synchrotron.param

```diff
@@ -0,0 +1,83 @@
+/* Copyright 2024 Filip Optolowicz
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ * This file contains parameters for synchrotron extension
+ */
+
+#pragma once
+
+namespace picongpu
+{
+    namespace particles
+    {
+        namespace synchrotron
+        {
+            namespace params
+            {
+                //! Turn off or turn on the electron recoil from electrons generated.
+                constexpr bool ElectronRecoil = true;
+
+                //! @todo bool for turning on or off the photon generation
```

Comment:
No, it would still take into account the recoil to the electron, but would not spawn a new photon macroparticle. (Ideal if you only want to take into account energy loss via radiation but not actually care for the x-ray photons emitted.)